Servers/UI/OJS.Servers.Ui.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -17,6 +17,11 @@ public static class ServiceCollectionExtensions
 
         private static readonly string ApiDocsTitle = $"{ApplicationFullName} {AppName} Api";
 
+        private static readonly string[] RequiredConfigValues =
+        {
+            EnvironmentVariables.UiHomeYouTubeVideoId,
```

Comment:
What is this variable? I don't know that we have HomeYouTubeVideo

Reply:
We are using it in index.cshtml. I added it because we don't have a validation check for it.

Reply:
If we are validating it now, how the app starts successfully if it's not in launchSettings?